_includes/cask.html

```diff
@@ -1,11 +1,15 @@
 
         <td>
-{%- if site.data.cask[include.sort_key] != nil -%}
-            <a href="{{ site.baseurl }}/cask/{{ include.cask.token | uri_escape }}">{{ include.cask.token | escape }}</a>
+{%- assign inc_c = site.data.cask[include.data_token] -%}
+{%- if inc_c != nil -%}
+            <a href="{{ site.baseurl }}/cask/{{ include.token | uri_escape }}">{{ include.token | escape }}</a></td>
+        <td>{{ inc_c.version | truncate: 25 | escape }}</td>
+        <td>{{ inc_c.desc | escape }}</td>
+        <td>{{ inc_c.name.first | escape }}
```

Comment:
```suggestion
{%- assign include_cask = site.data.cask[include.data_token] -%}
{%- if include_cask != nil -%}
            <a href="{{ site.baseurl }}/cask/{{ include.token | uri_escape }}">{{ include.token | escape }}</a></td>
        <td>{{ include_cask.version | truncate: 25 | escape }}</td>
        <td>{{ include_cask.desc | escape }}</td>
        <td>{{ include_cask.name.first | escape }}
```

unsure of any of these `include.token` should be `include_cask`?

Reply:
ok, thanks. do whatever you think is best :+1: